src/portalnet/utils.rs

```diff
@@ -41,26 +42,47 @@ mod test {
 }
 
 pub fn get_data_dir() -> String {
+    let path;
     match env::var(TRIN_DATA_ENV_VAR) {
-        Ok(data_path) => data_path,
-        Err(_) => get_default_data_dir(),
+        Ok(data_path) => {
+            path = data_path;
+        }
+        Err(_) => {
+            path = get_default_data_dir();
+        }
+    }
```

Comment:
It would be more rusty (rustic?) to do something like:

```
let path = env::var(TRIN_DATA_ENV_VAR).unwrap_or_else(get_default_data_dir);
```

Reply:
Damn I didn't even realise I could do that xD